packages/rbflabs-agoric-react-components/package.json

```diff
@@ -28,34 +28,22 @@
     "@agoric/eventual-send": "^0.14.1",
     "@agoric/notifier": "^0.4.0",
     "@agoric/web-components": "^0.2.0",
-    "@lit-labs/react": "^1.0.4",
-    "@types/node": "^17.0.42",
-    "@types/styled-components": "^5.1.25",
-    "eslint": "^8.17.0",
-    "lit": "^2.2.5",
     "react": "^18.0.0",
-    "react-dom": "^18.0.0",
-    "react-icons": "^4.4.0",
-    "react-toastify": "^9.0.3",
-    "styled-components": "^5.3.5",
-    "tsup": "5.10.1"
+    "react-dom": "^18.0.0"
+  },
+  "peerDependencies": {
+    "short-unique-id": "^4.4.4"
   },
   "devDependencies": {
-    "@originjs/vite-plugin-commonjs": "^1.0.3",
+    "@types/node": "^17.0.42",
     "@types/react": "^18.0.0",
     "@types/react-dom": "^18.0.0",
-    "@typescript-eslint/eslint-plugin": "^5.28.0",
-    "@typescript-eslint/parser": "^5.28.0",
-    "@vitejs/plugin-react": "^1.3.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "prettier": "^2.6.2",
+    "eslint": "^8.18.0",
+    "tsconfig-rbflabs": "*",
     "typescript": "^4.7.3",
-    "vite": "^2.9.9"
+    "tsup": "5.10.1"
```

Comment:
Why using `tsup` instead of `vite`?

Reply:
I did not introduce `tsup` into this. I only deleted `vite` as it was never used anywhere (including its config). I used it because it was already there.

I never used `tsup` before, but it seems to be simple enough to let it do its job for now and think about tooling whenever this repo expands.

Reply:
Oh, I see. We used Vite before but then we switched to Turborepo which probably came with this `tsup` and `vite` was never removed from here. Thanks for cleaning this up!